apps/web/src/services/dashboards/store/dashboard-detail-info.ts

```diff
@@ -290,9 +290,11 @@ const getConvertedWidgetLayouts = (storedWidgetLayouts: DashboardModel['layouts'
 
     return storedWidgetLayouts.map((layout) => layout.map((widgetInfo) => {
         const convertedInheritOptions = getConvertedWidgetInheritOptions(widgetInfo.inherit_options);
+        const convertedWidgetOptions = getConvertedWidgetOptions(widgetInfo.widget_options);
```

Comment:
nitpick: It seems like a NOTE is needed here too!

Reply:
This is the function that is only used in this file.
I added a note to the code that uses this function, so I think it is unnecessary..! :)